src/routes/console/project-[project]/messaging/providers/wizard/configure.svelte

```diff
@@ -82,6 +82,7 @@
             {:else if input.type === 'email'}
                 <InputEmail
                     id={input.name}
+                    isPopoverDefined={input.popover !== undefined}
```

Comment:
I don't think this PR is needed anymore since `isPopoverDefined` was added down below